writehat/components/base.py

```diff
@@ -469,6 +470,18 @@ def reviewStatusValue(self):
             return ReviewStatusField._choices['unassigned']
 
 
+    @property
+    def todoItems(self):
+
+        out = []
+        for k,v in self.validFields().items():
+            if getattr(v, "markdown", False):
+                match = re.search(todo_re, str(getattr(self, k, "")))
+                if match is not None:
+                    out.append(k)
```

Comment:
If a field has multiple todo items on it, would each one be a seperate todo item?
This should also apply to the todoItems method in finding.py

Reply:
The current function will only identify if there is a match for the `{todo}` regex or not; it should not repeat fields containing multiple tags. Is that a reasonable implementation?

Reply:
As it is now, it is working as intended I would say. Adding the ability to have multiple `{todo}` tags per field would allow for more flexibility I feel. An example would be using them as placeholders in templates. That way, it would be more obvious if there are still placeholder `{todo}` tags that need to be resolved. Being able to have multiple `{todo}` tags would also potentially allow viewing every active `{todo}` on the tooltip modal, rather then one at a time.

Reply:
Agreed. Multiple tags are supported, but the icon only functions to indicate whether or not a section has such a tag, not how many or how they're annotated. If users find themselves constantly refreshing to see if all tags have been addressed, we should expand the UI to better depict what tags remain. I expect we'll do that in the near future, but for now wanted to get the first iteration published to see how people use this feature before attempting to expand it.